en/installation/installation-of-a-poller/using-centreon-iso.md

```diff
@@ -123,6 +123,106 @@ Then restart your server with the following command:
 reboot
 ```
 
+## Register the server
+
+To register it to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <REMOTE_SERVER_NAME>
+```
+
+Example:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u admin -t Poller -h 192.168.0.1 -n poller-1
+```
+
+> Replace **<IP_CENTREON_CENTRAL>** by the IP of the Centreon server seen by the poller or by the Remote Server if you
+> want to link your server to it..
```

Comment:
I can't find where the software collection repository installation is explained for a poller
As php-cli-72 is required